db/migrate/20210722111142_remove_rhsm_columns.rb

```diff
@@ -0,0 +1,41 @@
+class RemoveRhsmColumns < ActiveRecord::Migration[6.0]
+  class Authentication < ActiveRecord::Base
+    self.inheritance_column = :_type_disabled
+  end
+
+  def up
+    change_table "miq_databases" do |t|
+      t.remove "registration_type"
+      t.remove "registration_organization"
+      t.remove "registration_server"
+      t.remove "registration_http_proxy_server"
+    end
+
+    Authentication.where(:authtype => %w[registration registration_http_proxy])
+                  .destroy_all
```

Comment:
This should be wrapped in a `say_with_time`.  Also, I'm thinking this should be moved into a separate migration so that it can be tested independently.

Reply:
Also this needs a spec.